### ISSUE TITLE
Requests | Suggestion | Ideas | Etc.

### ISSUE THREAD

**Issue by Marexn** [133spider/AQLite#37]
Hi there!
Been sparking up some ideas which could improve or at very least bring couple convenience options into game, and possibly get more players engaged.

I assume you've heard of Runelite, which I believe AQLite is kind of baste up on.
I would love to see the AQLite Settings be moved into a sidebar like with in Runelite, and to give the client a bit more polished look. I think it would attract more players to it.

One of the features I've been tackling on is the current problem with the lack of Wiki updates, or just how slowly they are been done, which is mostly due to there no longer being interest in investing lots of time and money to get items, and then write down all of the information and take pictures/gif's of the items etc, and due to there being no rewarding factor to it, other than the Wiki sword (RIP Harrison).

So, how can this be tackled?
Integrated Wiki plugin, which allows you to log into your Wiki account, and update/send update request for items on fly.
You could be able to press a button on Inventory after signing in to Wiki, which will automatically fill up information about of the item; Name, Description, Rarity, Type, AC/Member/Free, Price, Etc.
This would allow players to engage more with helping each others when they find new items and allow information of new releases to come about quicker. You could add picture to it, or have integrated screenshot plugin which could possibly search the swf file, and somehow make just png image of the item if possible.

Another suggestion i have is Social Media/News Tab.
This would allow you to follow on stuff posted by the community members, and you would be in charge on who you want to see in this tab, with possible exception to having AQLite's own Twitter fee always showing (Trynna find away for you to plug yourself more). You could have settings menu where you're able to add the @ of certain individuals, which will allow you to modify, mix and match your fee and personalize it to the way you want.

Another little thing that could be interesting to see is the message fee.
Now, currently in AQW, we can only see certain amount of messages until you aren't able to scroll up no more, and see your past/previous messages with someone. It would be great if there was away for you to go back on previous messages, and see what people might've talked about, while you were AFK. (I AFK every single day, and sometimes I do wonder what I've missed, so this would be kinda interesting.) Additionally, I would like to see message time stamps also. The time stamps could be baste on AQW's own server time.

Also, just little note. It would be great if the location I add to the Travel Menu would stay, even after i close the client.

These are just my personal ideas as of what would be nice to see added, and what could help the community and individuals.
The client has been a blessing so far mate honestly, and it has made playing this game so much more enjoyable, so from the bottom of me heart mate, thank ya'.

Of course, I would love to see other peoples ideas on how to improve some of my ideas and concepts.
Cheers 😊👍

**Comment by 133spider** [comment]
Hello,

- The Runelite theme for AQLite was what was I originally had planned, but I chose to have the settings ingame to keep the AQW style/theme. However, I can go back and experiment with that idea again.
- The Wiki Plugin is a great idea and I'm currently thinking of ways to implement it. However, I might have to figure out alternatives to automatically take "pictures" of said item
- I would need to do some research atm to see if a Social Media/News tab would be possible and not too demanding for the client
- Storing of messages is possible, but I'm going to have to take some time and figure out the UI for it
- I'll make sure to update the Travel Menu in the next version

**Comment by Marexn** [comment]
Yeah, I kinda figured where you got the idea from. The main reason i would see a side bar working is due to the poor implementation of the games size on AE's part. When you open the client to full window, there are still these black empty bars on both sides.

![image](https://user-images.githubusercontent.com/9413484/77907961-49ac2980-7293-11ea-80bf-65d82b4fe81f.png)

I thought that these could be reduced if you were able to create side bar which would push the black borders onto the either side to have the game at least look like it's not missing something.

**Comment by 133spider** [comment]
^ The black bars can be removed and they were initially removed, but because some players had insanely high screen resolutions it looked very off

You can eliminate the black bars by:
1. Going to AQLite\resources\app\public\index.html
2. Replace the keywords "default" with "exactfit" and save
(There are TWO "default" you have to change)
![image](https://user-images.githubusercontent.com/11169157/77994126-586bfe00-72de-11ea-91da-f4b022e383ea.png)

But yeah, I see what you mean by implementing the side bar to hide the black borders

**Comment by Marexn** [comment]
Okey yeah, I can see why some players felt strongly about of this. I feel as if everything is bit too stretchy and unnatural. I personally could not enjoy playing like this... But I think this could be helpful for some.

![image](https://user-images.githubusercontent.com/9413484/78147923-d8a87580-743c-11ea-8777-79f8b9f6a66a.png)

**Comment by MAAF72** [comment]
Can i merge AQLite with Grimoire and then release them to public?

Thanks before
